sh/tmuxcomplete

```diff
@@ -27,7 +27,8 @@
 # Words including 2000 lines of history per pane
 #     sh tmuxcomplete -c '-S -2000'
 
-if ! tmux info &> /dev/null; then
```

Comment:
@wellle  Not sure if it's the cause of the issue, but I am pretty sure `&>` is a bashism. The POSIX form would be `2>&1 >/dev/null`.